med-inventory/src/layout/NavBar.jsx

```diff
@@ -57,10 +56,7 @@ const useStyles = makeStyles(theme => ({
   menuButton: {
     marginRight: 13
   },
-  logoImage:{
-   height: "64px"
-  },
-  logoImageOpen:{
+  logoDrawerImageOpen:{
```

Comment:
Again, here its not image that is open. Use proper names going forward. Closing this for now